### PR TITLE
Update AdAdawy hosts issue report link

### PR DESCRIPTION
**Pull request by PerfectSlayer**
Hi @StevenBlack 

I hope you're going well.
I propose a small change about the link to report an issue about AdAway's default hosts. I link it to the site GitHub project, not the application.
Moreover, @jawz101 has taken over the `hosts` file management. There is [more frequent updates](https://github.com/AdAway/adaway.github.io/commits/master) than ever so you might also want to change the _occasionnaly_ frequence label.

I take the opportunity of this PR to announce your hosts will be integrated as default source for the AdAway application! 🎉 
I think some of the users already added it to their configuration but it will be enabled by default for all new users.
I hope it is fine to you. Don't hesitate to tell me if not.

Regards

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ URLHaus | A project from [abuse.ch](https://abuse.ch/) with the goal of sharing
...
 add.2o7Net | 2o7Net tracking sites based on [hostsfile.org](http://www.hostsfile.org/hosts.html) content. |[link](https://github.com/FadeMind/hosts.extras) | [raw](https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.2o7Net/hosts) | occasionally | GPLv3+ | [issues](https://github.com/FadeMind/hosts.extras/issues)
 KADhosts | Fraud/adware/scam websites. |[link](https://kadantiscam.netlify.com/) | [raw](https://raw.githubusercontent.com/PolishFiltersTeam/KADhosts/master/KADhosts_without_controversies.txt) | frequently | CC BY-SA 4.0 | [issues](https://github.com/PolishFiltersTeam/KADhosts/issues)
-AdAway | AdAway is an open source ad blocker for Android using the hosts file. |[link](https://adaway.org/) | [raw](https://raw.githubusercontent.com/AdAway/adaway.github.io/master/hosts.txt) | occasionally | CC BY 3.0 | [issues](https://github.com/AdAway/AdAway/issues)
+AdAway | AdAway is an open source ad blocker for Android using the hosts file. |[link](https://adaway.org/) | [raw](https://raw.githubusercontent.com/AdAway/adaway.github.io/master/hosts.txt) | occasionally | CC BY 3.0 | [issues](https://github.com/AdAway/adaway.github.io/issues)
```
Comment (funilrys) [inline readme.md:80]: Hi @PerfectSlayer! It's awesome to see the project being alive again 😊

Unfortunately, you made the change at the wrong place. All the readme are generated. 

Please apply the change (Only) to https://github.com/StevenBlack/hosts/blob/master/data/adaway.org/update.json!

Have a nice day/night.
Stay Healthy.

**Comment by PerfectSlayer** [comment]
Sorry for the trouble, I read the contributing guide but not the codebase structure document... It should be better. 

I didn't squash the changes as I don't want to loose your comment. Feel free to squash all three commits (original, revert then fix) when merging 😉 

Have a nice day too @funilrys !

**Comment by StevenBlack** [comment]
Thanks Bruce @PerfectSlayer!  Thanks for this PR, and the info in the comments above.

Merging!

**Comment by gallegonovato** [comment]
Good afternoon:



Many to @PerfectSlayer , @jawz101 and @StevenBlack for all your work and attention.    And to make this wonderful application even better. 

 

A hug

**Pull request merged**